### PR TITLE
Adds a better technical interview (2018)

### PR DESCRIPTION
**Pull request by JoshTGreenwood**

### CODE REVIEW

**Comment by searls** [comment]
FWIW I've been putting blog post titles in Sentence case for the last year or so. We aren't consistent about that but I'd like to be but I also don't want a :td: writing style guide so ¯\_(ツ)_/¯

File: app/posts/2018-05-04-a-better-technical-interview-part-2.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  url: "http://twitter.com/joshtgreenwood"
+video:
+  url: "https://vimeo.com/267312982"
+  type: "vimeo"
+reddit: false
+---
+
+Our interview processes tend to focus solely on one outcome: a successful hire. But what about all of the other people who go through our interview process? If we aren't thinking about them and empathizing with their situations, we are prone to giving out lots of bad experiences. Let's focus on every candidate who goes through our interview process and create great outcomes for everyone.
+
+The inspiration for this talk came from [Sam Jone's excellent talk of the same name](http://blog.testdouble.com/posts/2017-01-27-a-better-technical-interview).
```
Comment (stevenjackson) [inline app/posts/2018-05-04-a-better-technical-interview-part-2.md]: typo

**Comment by samjonester** [comment]
Since this post will have almost the same title as another, have you thought about merging the two?

**Comment by JoshTGreenwood** [comment]
I have no strong preference @samjonester. I'll do whatever @searls wants to do with it

**Comment by searls** [comment]
Since they're different performances of a fork of a talk, I don't really know what to do . I think keeping it titled "(2018)" and then linking prominently to Sam's original is probably best

**Pull request merged**